common/models/service-process.json

```diff
@@ -99,6 +99,11 @@
       "model": "ServiceMetric"
     }
   },
+  "indexes": {
+    "uniq_process": {
+      "keys": ["pid", "workerId", "startTime", "serviceInstanceId"]
```

Comment:
this is an extra compound index that is required to be unique? and its supported in all LB connectors, or least the ones we use? I wish that json could have comments :-(